index.html

```diff
@@ -57,82 +62,79 @@ <h1>WebGPU GPT Model Demo</h1>
       }
 
       async function startGeneration() {
-        setTextareaVisibility(true);
```

Comment:
so "visibility true" was a little confusing. It's really setting "visibility" to false